lerobot/configs/policy/act.yaml

```diff
@@ -4,7 +4,7 @@ seed: 1000
 dataset_repo_id: lerobot/aloha_sim_insertion_human
 
 override_dataset_stats:
-  observation.images.top:
+  observation.images.wrist:
```

Comment:
I suppose you plan to revert all of the changes in this file?

Reply:
Yes

Reply:
I'm actually not sure how to deal with the config for rlbench. For now, I have added a separate config file.